prow/cluster/jobs/istio/pkg/istio.pkg.master.yaml

```diff
@@ -0,0 +1,79 @@
+job_template: &job_template
+  branches:
+    - "^master$"
+  decorate: true
+  path_alias: istio.io/pkg
+
+istio_container: &istio_container
+  image: gcr.io/istio-testing/istio-builder:v20190624-478286dc
+  # Docker in Docker
+  securityContext:
+    privileged: true
+  resources:
+    requests:
+      memory: "2Gi"
+      cpu: "3000m"
+    limits:
+      memory: "24Gi"
+      cpu: "3000m"
+
+presubmits:
+
+  istio/pkg:
+
+    - name: pkg-lint-master
+      path_alias: istio.io/pkg
+      decorate: true
+      always_run: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-04-22
+            command:
+              - make
+              - lint
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: test-pool
+
+    - name: pkg-build-master
+      path_alias: istio.io/pkg
+      decorate: true
+      always_run: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-04-22
+            command:
+              - go
+              - build
+              - ./...
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: build-pool
+
+    - name: pkg-test-master
+      path_alias: istio.io/pkg
+      decorate: true
+      always_run: true
+      spec:
+        containers:
+          - image: gcr.io/istio-testing/circleci:2019-04-22
+            command:
+              - go
+              - test
+              - -race
+              - -coverprofile=coverage.txt
+              - -covermode=atomic
+              - ./...
+            env:
+              - name: GO111MODULE
+                value: "on"
+        nodeSelector:
+          testing: test-pool
+
+postsubmits:
```

Comment:
Remove?

Reply:
Removed, and removed in the operator's config too.